### ISSUE TITLE
Migrate to LLNL Organization?

### ISSUE THREAD

**Issue by IanLee1521** [davidbuttler/reconcile#1]
Hey David -- Is there any opposition to migrating this to the LLNL organization?

**Comment by davidbuttler** [comment]
This code was produced in collaboration with a university partner (U. Utah), so I don't want to claim full ownership under LLNL.  It is also a bit old and out of date -- I don't want to continue supporting it as there are likely better results (from Utah or elsewhere).

**Comment by IanLee1521** [comment]
Makes sense. The current link I had for it (on a list I got from @holdensanchez2) was https://gforge.cis.cornell.edu/projects/coref/ which is looking even older and more out of date than this one... Should I update the spreadsheet to point here instead?
